### PR TITLE
feat(CX-3276): add new route for mediam sale price

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat/Fix**

This PR solves [CX-3276]

### Description

Add a new route for median sale price
When assessing the page from collector profile, the URL should start with /collector-profile

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3276]: https://artsyproduct.atlassian.net/browse/CX-3276?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/e2e` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 83d402f23c9a2e788fca06bbb028dacdb25b25e9
</p>

**Review by MounirDhahri — APPROVED** [review]
LGTM

**Review by MrSltun — APPROVED** [review]
🌟

File: src/Apps/MyCollectionInsights/myCollectionInsightsCollectorProfileRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          ...MyCollectionInsightsMedianSalePriceAtAuction_artist
+        }
+      }
+    `,
+    cacheConfig: {
+      force: true,
+    },
+    // hideNav: true, // TODO: Hide/Unhide depending on the conversation with the #design team
+    // hideFooter: true, // TODO: Hide/Unhide depending on the conversation with the #design team
+  },
```
[the remark this answers is not in the document]
Comment (dariakoko) [reply src/Apps/MyCollectionInsights/myCollectionInsightsCollectorProfileRoutes.tsx:42]: Copied from [here](https://github.com/artsy/force/blob/48f1d450f808960f5db55759d0d3aa5e38ee011a/src/Apps/MyCollectionInsights/myCollectionInsightsRoutes.tsx)

[the remark this answers is not in the document]
Comment (dzucconi) [reply src/Apps/MyCollectionInsights/myCollectionInsightsCollectorProfileRoutes.tsx:42]: Rebased and updated the comment in my layouts PR

**Pull request merged**